### PR TITLE
Update Maxixi gear, adding in female set for some, and fixing some mo…

### PR DESCRIPTION
**Pull request by helixhamin**
…ds where wrong.

https://www.bg-wiki.com/bg/Maxixi_Attire_Set
https://www.bg-wiki.com/bg/Maxixi_Attire_Set_%2B1
https://www.bg-wiki.com/bg/Maxixi_Attire_Set_%2B2
https://www.bg-wiki.com/bg/Maxixi_Attire_Set_%2B3

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -26019,11 +26019,11 @@ INSERT INTO `item_mods` VALUES (23125,14,28);     -- CHR+28
 INSERT INTO `item_mods` VALUES (23125,23,23);     -- Attack+23
 INSERT INTO `item_mods` VALUES (23125,25,46);     -- Accuracy+46
 INSERT INTO `item_mods` VALUES (23125,29,6);      -- Magic Defense+6
-INSERT INTO `item_mods` VALUES (23125,31,72);     -- Magic Evasion+72
+INSERT INTO `item_mods` VALUES (23125,31,74);     -- ffxiah states Magic Evasion+74
 INSERT INTO `item_mods` VALUES (23125,68,69);     -- Evasion+69
 INSERT INTO `item_mods` VALUES (23125,384,40);    -- Haste +40
 INSERT INTO `item_mods` VALUES (23125,491,17);    -- Waltz Potency+17
-INSERT INTO `item_mods` VALUES (23125,497,1);     -- Waltz Recast+1
+INSERT INTO `item_mods` VALUES (23125,497,1);     -- Waltz Recast+1 (Should this be +1 or -1?)
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: https://github.com/DarkstarProject/darkstar/blob/824fbbd7c79eea6d43524c0d96131abffac56881/scripts/globals/abilities/curing_waltz.lua#L30

waltz recast doesn't truly exist yet. (hint: it shouldn't be happening there in the ability script anyway)

Comment (helixhamin) [inline sql/item_mods.sql]: So, recommended to just remove this, or just remove the comment in my markup?

Comment (TeoTwawki) [inline sql/item_mods.sql]: I'd not add the mod till its actually implemented.

Comment (TeoTwawki) [inline sql/item_mods.sql]: if it was already there the comment you did can stay (my eyes are not liking how githubs red looks like light pink on a  white background lately)

**Comment by helixhamin** [comment]
@TeoTwawki Thanks, I removed that mod for now. Guess I will need to wait until someone creates the actual mod. I also standardized the comments, since as you noted in the other PR, comments should be standard in the file. My bad, I will stick the comments in the PR itself, not the sql files.

**Pull request merged**